parsec/core/fs/fs.py

```diff
@@ -66,60 +66,76 @@ def _get_workspace(self, path):
         return workspace, subpath
 
     async def stat(self, path: str) -> dict:
+        # XXX: This method should be splitted in several methods:
```

Comment:
better use `TODO` everywhere to simplify grep

Reply:
Alright I'll fix it